docs/src/components/FancyLink.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { Figma, Github, ReactJs, Spectrum, Storybook } from "@icons-pack/react-simple-icons";
+import { ColorPicker, Download, FlightDirect, Google, Tips } from "@kiwicom/orbit-components/icons";
+import { Tile } from "@kiwicom/orbit-components";
+
+import useIsUrlExternal from "../hooks/useIsUrlExternal";
+
+interface FancyLinkProps {
```

Comment:
no need to name it specifically `FancyLinkProps`, there is only one component here, so just `Props` as we usually do with components :)